overthebox/files/usr/share/otb/post-tracking.d/get-ip-asn

```diff
@@ -15,7 +15,13 @@ _curl() {
 }
 
 pub_ip=$(_curl ifconfig.ovh)
```

Comment:
You forgot all the checks about curl output ( check that it's not empty or 'null' ) before doing anything with the value